.github/workflows/ci.yml

```diff
@@ -45,8 +45,7 @@ jobs:
       env:
         CODACY_PROJECT_TOKEN: ${{ secrets.CODACY_TOKEN }}
       run: |
-        pip install codacy-coverage
-        python-codacy-coverage -r coverage.xml
```

Comment:
This repo is archived and marked as deprecated in the README https://github.com/codacy/python-codacy-coverage

I wonder if this may be related to an intermittent (?) 404 when uploading coverage reports

```
requests.exceptions.HTTPError: 404 Client Error: Not Found for url: https://api.codacy.com/2.0/coverage/b98920800d09f380675b1bacd8ce62e56f0e1c8a/python
```

Hopefully this updates us and smooths that issue out at the same time.

Reply:
This comment is now moot.